### DISCUSSION TITLE
The --disable-notifications flag should be called --disable-events

### DISCUSSION THREAD

**Discussion by VBrueggemann** [spatie/laravel-backup#1096] [General] [answered]
Out of the box `disable-notifications` and `disable-events` are basically the same.
But when an Event Listener is added that is unrelated to notifications (e.g. encryption ) it might cause confusion.

**Comment by freekmurze** [comment, marked as answer]
Feel free to suggest concrete improvements.

I'm going to leave things as they are for now.
